app/api/datastore/entities/maat/application.rb

```diff
@@ -2,13 +2,48 @@ module Datastore
   module Entities
     module Maat
       class Application < Grape::Entity
+        expose :id
+        expose :schema_version
         expose :reference
-        expose :applicant
+        expose :submitted_at, format_with: :iso8601
+        expose :date_stamp
+        expose :client_details
+        expose :provider_details
+        expose :case_details
+        expose :interests_of_justice
+        expose :ioj_passport
 
         private
 
-        def applicant
-          object.application&.dig('client_details', 'applicant')
+        def client_details
+          object.application&.dig('client_details')
```

Comment:
Thining we could have another private method like:

```ruby
def application_value(name)
  object.application&.dig(name)
end
```

Just to remove the repitition of calling `object.application&.dig()` multiple times.

Reply:
Good shout